packages/terra-demographics-banner/src/DemographicsBannerDisplay.jsx

```diff
@@ -62,7 +62,7 @@ const propTypes = {
   /**
    * Photo to display in the banner
    */
-  photo: PropTypes.node,
+  photo: PropTypes.string,
```

Comment:
this is non-passive. Are we planning to release this as a MVB?

Reply:
cc: @ShettyAkarsh @ryanthemanuel

Reply:
Changed back to Node to keep Avatar changes as passive to existing demographics banner